### DISCUSSION TITLE
Brightness level not restored correctly (LG UltraFine 5K, MacBook Pro M1 Max)

### DISCUSSION THREAD

**Discussion by webdeck** [MonitorControl/MonitorControl#874] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

When waking from sleep, the brightness of my 2 x LG UltraFine 5K displays is not always restored.  I have tried all three settings for the "upon startup or wake" preference, and the issue happens with each of them.

### Steps to reproduce

1. Sleep Mac
2. Wake Mac
3. May need to repeat multiple times

Monitor will wake with brightness dimmer than before it was put to sleep.  If you press the dimmer key, the monitor jumps up to actually be brighter (i.e. one step below where it was when the mac was put to sleep.)

### Expected behavior

Monitor wakes at same brightness level as when it was put to sleep.

### Anything else?

After wake, the menu shows the brightness value as what it was before sleep, even though the monitor is much dimmer than the displayed value.

This has become my ritual when waking my Mac now - press dimmer key then brighter key after waking to restore correct brightness level.

I have the laptop in clamshell mode.

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.1 Monterey
- Mac model: MacBook Pro M1 Max 16"
- MonitorControl version: 4.0.2 build 6965
- Monitor(s): 2 x LG UltraFine 5K
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment, marked as answer]
Hi, the UltraFine 5K is controlled natively by macOS (MC also uses Apple's APIs to control the display). Restoring brightness applies only to DDC displays.
